routers/api/v1/repo/repo.go

```diff
@@ -173,7 +173,7 @@ func CreateOrgRepo(ctx *context.APIContext, opt api.CreateRepoOption) {
 	//       403: forbidden
 	//       500: error
```

Comment:
You need to update the swagger:route (the url part) also. You can even edit {org} since in url params in swagger at not yet configured.